sound/soc/sof/intel/cnl.c

```diff
@@ -398,6 +398,7 @@ const struct sof_intel_dsp_desc cnl_chip_info = {
 	.sdw_alh_base = SDW_ALH_BASE,
 	.check_sdw_irq	= hda_common_check_sdw_irq,
 	.check_ipc_irq	= hda_dsp_check_ipc_irq,
+	.dsp_init = cl_dsp_init,
```

Comment:
commit message

ASoC: SOC: Intel: introduce cl_init callback

The code loader init sequences are different between versions of Intel platforms.
Have a cl_init callback allows us to reuse the common code.
No function changed.